onnxruntime/core/flatbuffers/flatbuffers_utils.cc

```diff
@@ -95,6 +95,22 @@ static Status SaveTensorTypeAndShapeOrtFormat(flatbuffers::FlatBufferBuilder& bu
   return Status::OK();
 }
 
+#if !defined(ORT_MINIMAL_BUILD)
+static Status SaveSparseTensorTypeOrtFormat(flatbuffers::FlatBufferBuilder& builder,
```

Comment:
Better to completely comment it out saying we don't currently support sparse tensor as a node input/output so don't need to handle serializing its type info.

This code is already inside an ORT_MINIMAL_BUILD ifdef though from line 20 (all the 'save' logic is excluded in a minimal build as we only load models)